.github/workflows/release-please-v1.yml

```diff
@@ -81,9 +90,59 @@ jobs:
           echo "token=$token" >> "$GITHUB_OUTPUT"
 
       - name: Run release-please
+        if: "! inputs.autorelease"
         id: release_please
         uses: googleapis/release-please-action@v4
         with:
           token: ${{ steps.app_token.outputs.token }}
           config-file: release-please-config.json
           manifest-file: .release-please-manifest.json
+
+      - name: Write .releaserc
+        if: inputs.autorelease
+        run: |
+          echo '{
```

Comment:
this sucks, I can't find a better way: https://github.com/orgs/community/discussions/63863